### ISSUE TITLE
[Bug] Telegram apps not opening

### ISSUE THREAD

**Issue by CanerKaraca23** [Dr-TSNG/ZygiskOnKernelSU#162]
### Steps to reproduce/复现步骤

Open any telegram app/bot like Blum

### Expected behaviour/预期行为

Telegram app/bot opening.

### Actual behaviour / 实际行为

Telegram app/bot not opening.

### Module List / 模块列表

```Shell
Adguard Certificate
Play Integrity Fix
build.prop optimizer
Task profiles for uclamp
Tricky Store
VBMeta Fixer
YouTube Revanced
Shamiko
Zygisk Assistant
zygisk-detach
Zygisk Next
```

### Zygisk Next version / Zygisk Next 版本

1.2.4

### Android version / Android 版本

14

### Magisk/KernelSU/APatch version / Magisk/KernelSU/APatch 版本

KernelSU 0.9.5, KSU Manager 1.0.1

### Version requirement / 版本要求

- [X] I am using latest version of Zygisk Next/我正在使用最新版本 Zygisk Next

### Logs/日志

[KernelSU_bugreport_2024-11-14_03_14.tar.gz](https://github.com/user-attachments/files/17740785/KernelSU_bugreport_2024-11-14_03_14.tar.gz)

**Comment by CanerKaraca23** [comment]
![Screenshot_20241114-030825_Telegram](https://github.com/user-attachments/assets/2b29a850-5dd4-492f-928f-3565ce7352e4)

It stuck like this.

**Comment by 5ec1cff** [comment]
Disable Zygisk Assistant

**Comment by CanerKaraca23** [comment]
> Disable Zygisk Assistant

Unfortunately, still same.

Zygisk Next 1.1.0 was working fine. Downgrading ZN fixes this problem. Maybe is there a problem  ZN with System WebView?

**Comment by 5ec1cff** [comment]
Disable Zygisk Assistant, Shamiko and ksu's umount for Telegram and retry

**Comment by CanerKaraca23** [comment]
> Disable Zygisk Assistant, Shamiko and ksu's umount for Telegram and retry

Yes, Shamiko is the problem, thanks. Zygisk Assistant and KSU umount fine, no problem. But, Shamiko + ZN 1.1.0 also works fine. I will probably downgrade to 1.1.0 until you fix the problem. @5ec1cff

**Comment by 5ec1cff** [comment]
Please run `strace -y -s 1024 -p $(pidof webview_zygote) -f >/data/local/tmp/strace.log 2>&1` , launch any webview (e.g. The Telegram app you mentioned), wait for some seconds and send /data/local/tmp/strace.log

You need to keep zygisk modules other than shamiko disabled to do this

If you don't have strace you can use this:

[strace.zip](https://github.com/user-attachments/files/17745504/strace.zip)

**Comment by CanerKaraca23** [comment]
> Please run `strace -y -s 1024 -p $(pidof webview_zygote) -f >/data/local/tmp/strace.log 2>&1` , launch any webview (e.g. The Telegram app you mentioned), wait for some seconds and send /data/local/tmp/strace.log
> 
> You need to keep zygisk modules other than shamiko disabled to do this
> 
> If you don't have strace you can use this:
> 
> [strace.zip](https://github.com/user-attachments/files/17745504/strace.zip)

How can i use/install this strace?

I got permission denied error.

**Comment by 5ec1cff** [comment]
> I got permission denied error.

chmod +x

It's suggested to put the file under /data/local/tmp

**Comment by CanerKaraca23** [comment]
> > I got permission denied error.
> 
> chmod +x
> 
> It's suggested to put the file under /data/local/tmp

Not working 😥 @5ec1cff 

![Screenshot_20241114-191806_Termux](https://github.com/user-attachments/assets/ed1e131e-9395-4181-a891-3e558d5d5cff)

**Comment by 5ec1cff** [comment]
You forgot `su`

**Comment by CanerKaraca23** [comment]
> You forgot `su`

- I put your strace file to my /data/local/tmp
- I gave all permissions to termux
- `su`
- `chmod +x`
- `strace -y -s 1024 -p $(pidof webview_zygote) -f >/data/local/tmp/strace.log 2>&1`

This error. What I'm doing wrong?

strace.log says "/system/bin/sh: strace: inaccessible or not found"

@5ec1cff 

![Screenshot_20241114-194719_Termux](https://github.com/user-attachments/assets/48246949-e9e0-43d9-950f-011b14b9ff83)

**Comment by 5ec1cff** [comment]
You didn't even search how to use chmod

chmod +x /data/local/tmp/strace

**Comment by 5ec1cff** [comment]
No need to use strace, just type `ps -ef | grep webview_zygote` and show your result

**Comment by CanerKaraca23** [comment]
> You didn't even search how to use chmod
> 
> chmod +x /data/local/tmp/strace

I did this, .log file says "/system/bin/sh: strace: inaccessible or not found"

**Comment by CanerKaraca23** [comment]
> No need to use strace, just type `ps -ef | grep webview_zygote` and show your result

![Screenshot_20241114-195753_Termux](https://github.com/user-attachments/assets/b4e3f33e-c6d1-480e-b985-9c1195b33ed4)

**Comment by 5ec1cff** [comment]
replace `strace` with `/data/local/tmp/strace`

**Comment by CanerKaraca23** [comment]
> replace `strace` with `/data/local/tmp/strace`

You mean put strace to this directory? I already putted. @5ec1cff 

I'm really sorry for bothering you. I stole your time, sorry, I'm inexperienced about these things. My Telegram is @caner_karaca if you want to contact me from here, i spammed here.

![Screenshot_20241114-200026_MT_Manager](https://github.com/user-attachments/assets/1d06c200-39eb-419d-b059-ba598e5b4264)

**Comment by real-LiHua** [comment]
```shell
/data/local/tmp/strace -y -s 1024 -p $(pidof webview_zygote) -f >/data/local/tmp/strace.log 2>&1
```

**Comment by Genxster1998** [comment]
+1 facing similar issue of crashing webview_zygote on r8q . Will try to provide strace log with other modules disabled soon.
